Lib/fontTools/cffLib/CFF2ToCFF.py

```diff
@@ -69,14 +76,15 @@ def _convertCFF2ToCFF(cff, otFont):
                 if hasattr(privateDict, key):
                     delattr(privateDict, key)
 
+    cff.desubroutinize()
```

Comment:
So downgrading requires to de-subroutinize (remove and expand subroutines), and the resulting CFF 1.0 is only 'specialized' but not fully subroutinized. Maybe we should make that clearer in the --help section

Reply:
I swear somewhere I had written code to calculate stack use of a charstring. But can't find it :((